README.md

```diff
@@ -45,86 +42,105 @@ Get the following:
-```
+   ```bash
+   docker pull ghcr.io/hackyourcareer/smartkickers-frontend:latest
+   ```
 
-4. Pull backend image from GCR
+4. Pull the backend image from GCR.
 
-```bash
-docker pull ghcr.io/hackyourcareer/smartkickers-backend:latest
-```
+   ```bash
+   docker pull ghcr.io/hackyourcareer/smartkickers-backend:latest
+   ```
 
 ## Launching
 
-> **NOTE:** Configuration files for the camera view are provided by the developers in the [`SmartKickersAI`](https://github.com/HackYourCareer/SmartKickersAI/tree/main/LocalServer) repository.
+> **NOTE:** The configuration files for the camera view are provided by the developers in the [`SmartKickersAI`](https://github.com/HackYourCareer/SmartKickersAI/tree/main/LocalServer) repository.
 
 ### Launching the server and web application
 
-- Run pulled containers
+Run the pulled containers to launch the server and web application.
 
-  - In a terminal run:
+1. In a terminal run:
 
-  ```bash
-  docker run -p 3000:3000 ghcr.io/hackyourcareer/smartkickers-backend:latest
-  ```
+   ```bash
+   docker run -p 3000:3000 ghcr.io/hackyourcareer/smartkickers-backend:latest
+   ```
 
-  - Open a new terminal and run:
+2. Open a new terminal and run:
 
-  ```bash
-  docker run -p 3007:80 ghcr.io/hackyourcareer/smartkickers-frontend:latest
-  ```
+   ```bash
+   docker run -p 3007:80 ghcr.io/hackyourcareer/smartkickers-frontend:latest
+   ```
 
 ### Launching the ball-tracking software
 
 1. Plug in the camera to the computer.
 
-2. In order to launch the application, navigate to the `SmartKickersAI` repo and double click `start` file.
-
-   Successful connection to the Go server shows the connection details like in the screenshot below:
-
-   ![Node launch image](assets/nodeLaunch.png "Node launch")
-
-3. In your Internet browser, go to [`localhost:3007`](http://localhost:3007/) to see the React application.
-
-This takes you to the SmartKickers React application.
-
-![React application](assets/reactApp.png "React application")
+2. To launch the application, navigate to the `SmartKickersAI` repo and double click the `start` file.
+
+   On successful connection to the Go server, you see the following output:
+
+   ```bash
+    {
+      url: 'ws://127.0.0.1:3000',
+      wsurl: 'ws://127.0.0.1:3000/shot',
+      dispatcherlURL: 'ws://127.0.0.1:3000',
+      cfURL: 'ws://127.0.0.1:3000/cf',
+      tableID: '10',
+      token: 'c3d8c29ec43b9eadb8bc80ad1458ab8',
+      messageTypes: {
+          BallPosition: '615eec77d6be09356891',
+          Shot: '2ca34aee4bb3dec060b1',
+          BestShotGuid: 'c01b38461a678f3eefa5'
+      }
+   }
+   init
+   conn!
+   { type: 'utf8', utf8Data: '{"start":"10"}' }
+   { start: '10' }
+   game id
+   10
+   connected to IoT Services via wss
+   ```
+
+3. In your Internet browser, go to [`localhost:3007`](http://localhost:3007/) to open the React application.
+
+   You can now play the Smart Kickers game. Enjoy!
+
+   ![React application](assets/reactApp.png "React application")
```

Comment:
```suggestion
You can now play the Smart Kickers game. Enjoy!

![React application](assets/reactApp.png "React application")
```

These actually shouldn't be indented, as they describe the result of running the whole guide, or at least the whole section, right? 
I mean, it works this way as well, because that's also what the final command does, but we put it here to close this whole section, so we should want to make it general.